QUARKUS-711.md

```diff
@@ -0,0 +1,22 @@
+# QUARKUS-711 - RESTEasy support for Multipart data
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-711
```

Comment:
Noticed one thing I missed yesterday, Quarkus extension artifactIds start with `quarkus-...`.
Please adjust the text:
```
resteasy-multipart => quarkus-resteasy-multipart
resteasy => quarkus-resteasy
```